From 88573470dd0a3e6d02a4b52f5b35124aad271436 Mon Sep 17 00:00:00 2001
From: Yuwei Chen <yuwei.chen@intel.com>
Date: Fri, 24 Jul 2020 10:25:52 +0800
Subject: [PATCH] Test case for Driver - IdtVectorAsm.nasm file

Description: Change the address in the MdeModulePkg/Core/DxeIplPeim/Ia32/IdtVectorAsm.nasm

Repo: edk2
Signed-off-by: Yuwei Chen <yuwei.chen@intel.com>
---
 MdeModulePkg/Core/DxeIplPeim/Ia32/IdtVectorAsm.nasm | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/MdeModulePkg/Core/DxeIplPeim/Ia32/IdtVectorAsm.nasm b/MdeModulePkg/Core/DxeIplPeim/Ia32/IdtVectorAsm.nasm
index 4f9b98f18b..42cf6ac337 100644
--- a/MdeModulePkg/Core/DxeIplPeim/Ia32/IdtVectorAsm.nasm
+++ b/MdeModulePkg/Core/DxeIplPeim/Ia32/IdtVectorAsm.nasm
@@ -37,7 +37,7 @@ ASM_PFX(AsmGetVectorTemplatInfo):
 
 global ASM_PFX(AsmVectorFixup)
 ASM_PFX(AsmVectorFixup):
-        mov   eax, dword [esp + 8]
+        mov   eax, dword [esp + 7]
         mov   ecx, [esp + 4]
         mov   [ecx + (@VectorNum - @VectorTemplateBase)], al
         ret
-- 
2.27.0.windows.1

